org-code-javabuilder/protocol/src/main/java/org/code/protocol/JavabuilderThrowableMessageUtils.java

```diff
@@ -68,7 +80,7 @@ public static String getLoggingString(Throwable throwable) {
   private static String getDefaultFallbackMessageString(Throwable cause) {
     String loggingString = "Exception message: " + getExceptionMessageString(cause) + "\n";
     loggingString += getUserFacingStackTraceString(cause);
-    return loggingString;
+    return trimDetail(loggingString, MAX_DETAIL_SIZE_CHARS);
```

Comment:
nit: I think we typically have a preference to use `this`, i.e. `this.trimDetail()`?

Reply:
It's a static method so there's no `this`, right?

Reply:
Oh whoops my bad, missed that it was static :)